monai/networks/nets/swin_unetr.py

```diff
@@ -320,8 +320,8 @@ def _check_input_size(self, spatial_shape):
             )
 
     def forward(self, x_in):
-        if not torch.jit.is_scripting():
-            self._check_input_size(x_in.shape[2:])
+        # if not torch.jit.is_scripting() and not torch.jit.is_tracing():
+        #    self._check_input_size(x_in.shape[2:])
```

Comment:
May I know why comment these 2 lines?
@tangy5 Could you please help confirm this change?

Thanks.

Reply:
Oh those I have commented out while trying dynamo onnx export - I will revert it.